docs/configuration/index.md

```diff
@@ -631,6 +631,37 @@ Store task logs in HDFS. Note that the `druid-hdfs-storage` extension must be lo
 |`druid.indexer.logs.kill.initialDelay`| Optional. Number of milliseconds after Overlord start when first auto kill is run. |random value less than 300000 (5 mins)|
 |`druid.indexer.logs.kill.delay`|Optional. Number of milliseconds of delay between successive executions of auto kill run. |21600000 (6 hours)|
 
+### API Error Response
+
+Druid API error responses can be configured to hide internal information (such as Druid class name, stack trace, thread name, servlet name, code, line/column number, and host/ip). 
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.server.http.showDetailedJettyErrors`|When set to true, any error resulted from the Jetty layer / Jetty filter will have `servlet`, `message`, `url`, `status` and (if exist) `cause` fields in the JSON response. When set to false, the JSON response will only contains `message`, `url`, and `status`. Note that the value of the fields remain unchanged.|true|
+|`druid.server.http.errorResponseTransformStrategy.strategy`|Error response transform strategy to use to transform error response produced within Druid services.|`none`|
+
+##### Error response transform strategy
+
+Error response transform strategy can be use to transform error response produced within Druid services.
+Currently, when error response transform strategy is used (not `none`), the error responses from the following Druid services will be transformed as follow:
```

Comment:
```suggestion
When you specify an error response transform strategy other than `none`, Druid transforms the error responses from Druid services as follows:
```

Reply:
is this always true for all strategies or just true for all implemented strategies?

Reply:
@techdocsmith Done
@clintropolis This is always true for all strategies. Which field is included, removed, transformed is dependent on the Exception class. _How_ the error message is transformed depends on the strategy.